### PR TITLE
Switch to current catch2 release

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Signed-off-by: Alan Jowett <alanjo@microsoft.com>

### CODE REVIEW

File: tests/xdp/packages.config [hunk 1]
```diff
@@ -1,8 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
```
Comment (dthaler) [inline tests/xdp/packages.config:2]: Why was this comment removed?  I think this change should be reverted.

**Review by dthaler — APPROVED** [review]

File: tests/xdp/packages.config [hunk 2]
```diff
@@ -1,8 +1,4 @@
...
-  Copyright (c) Microsoft Corporation
-  SPDX-License-Identifier: MIT
--->
 <packages>
```
Comment (shankarseal) [inline tests/xdp/packages.config:2]: why was this file moved?

Comment (Alan-Jowett) [reply tests/xdp/packages.config]: Goal is to get rid if it, but this test has a hard dependency on Catch 2.8.0

Comment (shankarseal) [reply tests/xdp/packages.config]: which part? The custom argument? Please create an issue suggesting alternate so that the reference can be removed.

Comment (Alan-Jowett) [reply tests/xdp/packages.config]: https://github.com/microsoft/ebpf-for-windows/issues/820

**Review by shankarseal — APPROVED** [review]
